packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md

```diff
@@ -188,6 +188,14 @@ There is also an optimization where a state can be supported by `n` signatures o
 
 In the extreme, this allows a single state signed by all `n` parties to be accepted by the chain.
 
+:::note
+In most cases where a support proof is required for some change of state of the chain, the entire proof is submitted in the call data, and no on chain `State` is involved. The [`respond`](./clear-a-challenge#call-respond) method is an exception to this rule, and allows for the submission of only a single state in certain circumstances, with the support proof being implied by a combination of on-chain and submitted data.
```

Comment:
> the entire proof is submitted in the call data
I'd keep the term "call data" as one word, `calldata`

> no on chain `State` is involved
You should use the term "on-chain storage"

> a combination of on-chain and submitted data.
A combination of on-chain storage and data provided in calldata

Reply:
> You should use the term "on-chain storage"

To be precise, on-chain storage is checked (`largestTurnNum`). It's just that no validTransition check is done.